packages/altive_lints/lib/src/lints/avoid_single_child.dart

```diff
@@ -76,7 +76,18 @@ class AvoidSingleChild extends DartLintRule {
         if (childrenList.elements.length != 1) {
           return;
         }
+        for (final element in childrenList.elements) {
+          if (element is IfElement) {
+            if (_hasMultipleChild(element.thenElement)) {
+              return;
+            }
 
+            if (element.elseElement != null &&
+                _hasMultipleChild(element.elseElement!)) {
```

Comment:
![badge](https://img.shields.io/badge/review-imo-important)
`! ` as little as possible. Do you know why?

```suggestion
            if (element.elseElement case final CollectionElement ce
                when _hasMultipleChild(ce)) {
```